### PR TITLE
Basic docs for Geometry plus two new functions

### PR DESCRIPTION
**Pull request by ghost**
Basic docs and you can now call `Geometry. convex_hull_2d` and `Geometry. clip_polygon` from gdscript

### CODE REVIEW

File: doc/classes/Geometry.xml [hunk 1]
```diff
@@ -15,6 +15,7 @@
 			<argument index="0" name="extents" type="Vector3">
 			</argument>
 			<description>
+				Returns an array with 6 [code]Plane[/code]s that describe the sides of a box centered at the origin. The box size is defined by [code]extents[/code], which represents one (positive) corner of the box (i.e. half its actual size).
```
Comment (Noshyaar) [inline doc/classes/Geometry.xml]: For types we use `[Plane]`.

Comment (poke1024) [reply doc/classes/Geometry.xml]: Changed accordingly.

File: core/bind/core_bind.cpp [hunk 2]
```diff
@@ -1316,6 +1316,17 @@ Vector<int> _Geometry::triangulate_polygon(const Vector<Vector2> &p_polygon) {
 	return Geometry::triangulate_polygon(p_polygon);
 }
 
+Vector<Point2> _Geometry::convex_hull_2d(const Vector<Point2> &p_points) {
+
+	return Geometry::convex_hull_2d(p_points);
+}
+
+Vector<Vector3> _Geometry::clip_polygon(const Vector<Vector3> &p_points, const Plane &p_plane)
+{
```
[the remark this answers is not in the document]
Comment (poke1024) [reply core/bind/core_bind.cpp]: :see_no_evil:

**Review by ghost — APPROVED** [review]

**Pull request merged**